src/posix/platform/netif.cpp

```diff
@@ -309,10 +649,15 @@ static void processNetifAddrEvent(otInstance *aInstance, struct nlmsghdr *aNetli
         case IFA_LOCAL:
         case IFA_BROADCAST:
         case IFA_ANYCAST:
+        case IFA_MULTICAST:
```

Comment:
Have you checked whether the linux kernel will issue link events on multicast join? Based on my previous experience on wpantund and platform netif, it won't and that's why we need the mld listener.

Reply:
You are correct that it does not presently.  I had added this case during some debugging, as it seems correct -- if we did get a multicast address, then this code would handle it properly.  But the MLD listener would not be needed in that case.  Have you seen the other flavors of addressing here?  Or only IFA_ADDRESS?

Reply:
In fact I've only found IFA_ADDRESS and IFA_LOCAL.